### PR TITLE
HBASE-22981 Remove unused flags for Yetus

### PR DESCRIPTION
**Pull request by petersomogyi**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| :blue_heart: |  reexec  |   0m 35s |  Docker mode activated.  |
||| _ Prechecks _ |
| :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| :blue_heart: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 35s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| :blue_heart: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| :green_heart: |  shellcheck  |   0m  3s |  There were no new shellcheck issues.  |
| :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
||| _ Other Tests _ |
| :blue_heart: |  asflicense  |   0m  0s |  ASF License check generated no output?  |
|  |   |   2m 21s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-591/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/591 |
| Optional Tests | dupname asflicense shellcheck shelldocs |
| uname | Linux 30934c211ed0 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-591/out/precommit/personality/provided.sh |
| git revision | master / fbd5b5e327 |
| Max. process+thread count | 52 (vs. ulimit of 10000) |
| modules | C:  U:  |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-591/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) shellcheck=0.4.4 |
| Powered by | Apache Yetus 0.11.0 https://yetus.apache.org |


This message was automatically generated.

File: dev-support/hbase_nightly_yetus.sh [hunk 1]
```diff
@@ -67,6 +64,7 @@ YETUS_ARGS=("--whitespace-tabs-ignore-list=${WHITESPACE_IGNORE_LIST}" "${YETUS_A
 YETUS_ARGS=("--sentinel" "${YETUS_ARGS[@]}")
 YETUS_ARGS=("--branch=${BRANCH_NAME}" "${YETUS_ARGS[@]}")
 YETUS_ARGS=("--tests-filter=${TESTS_FILTER}" "${YETUS_ARGS[@]}")
+YETUS_ARGS=("--ignore-unknown-options=true" "${YETUS_ARGS[@]}")
```
Comment (busbey) [inline dev-support/hbase_nightly_yetus.sh]: What's still unknown?

Comment (petersomogyi) [reply dev-support/hbase_nightly_yetus.sh]: JDK/Hadoop specific stages do not run all plugins so whitespace, author and shelldocs are not recognized options there.

Comment (busbey) [reply dev-support/hbase_nightly_yetus.sh]: Okay that makes sense.

**Pull request merged**